src/main/java/core/basesyntax/service/impl/DataConverterImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.DataConverter;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> inputList) {
+        inputList.remove(0);
+        return inputList.stream()
+                 .map(s -> s.split(","))
+                 .map(s -> new FruitTransaction(FruitTransaction.Operation.getFromChar(s[0]),
+                                                s[1], Integer.parseInt(s[2])))
+                 .toList();
+    }

```

Comment:
Consider adding error handling for potential issues such as incorrect file format, invalid operation types, or invalid quantity values. Right now, if any of these issues occur, the method will throw an exception which might not be informative for the end-user. You could catch these exceptions and rethrow them with a more informative message.